packages/opentelemetry-exporter-prometheus/src/PrometheusExporter.ts

```diff
@@ -33,13 +33,15 @@ export class PrometheusExporter implements MetricExporter {
     port: 9464,
     endpoint: '/metrics',
     prefix: '',
+    appendTimestamp: true,
   };
 
   private readonly _logger: api.Logger;
   private readonly _port: number;
   private readonly _endpoint: string;
   private readonly _server: Server;
   private readonly _prefix?: string;
+  private readonly _appendTimestamp?: boolean;
```

Comment:
Nit: is `?` needed here ?

Reply:
I don't think so. Just removed it. Thanks, @obecny.